<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiComponentEditor</class>
 <widget class="QWidget" name="uiComponentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="8" column="1" colspan="3">
    <widget class="QComboBox" name="uiStatus"/>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="QTextEdit" name="uiLink"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="uiComment"/>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="uiType">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="uiNumberID">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="uiDescription"/>
   </item>
   <item row="9" column="3">
    <widget class="QSpinBox" name="uiQuantityPackage"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Per Unit:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="uiQuantity"/>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QStackedWidget" name="uiManufacture">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="uiManufacturePage1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="noneditable">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiManufacturePage2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="editable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Seller:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="uiName"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Manufacture:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLineEdit" name="uiSeller"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Link:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="QLineEdit" name="uiPrice"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
